README.md

```diff
@@ -33,11 +33,14 @@ invoking Packer directly. You can initiate the build process by running the
 following command in the root of this repository:
 
 ```bash
-# build an AMI with the latest Kubernetes version
+# build an AL2 AMI with the latest Kubernetes version
```

Comment:
now that we've built the story for multiple os distros, should we just say default?
```suggestion
# build an AMI with the latest Kubernetes version and the default OS distro
```